x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch/chart_switch.scss

```diff
@@ -24,6 +24,15 @@
   display: inline-flex;
 }
 
+.lnsChartSwitch__option {
+  .euiSelectableListItem__text {
+    flex-grow: 0;
+  }
+  .euiSelectableListItem__append {
+    margin-left: $euiSizeXS;
+  }
+}
```

Comment:
These style overrides are causing the selectable list's "enter" key badge that appears on focus to be in the wrong position. This focus badge should be aligned right, but this change causes it to appear to the left and immediately adjacent to the selectable list text content.

![CleanShot 2024-03-14 at 16 22 26](https://github.com/elastic/kibana/assets/3884767/4b296285-e045-4575-84af-fdf7b01360ae)

Would it be possible to fix this? My first thought would be to try moving the technical preview badge and warning icon into the `.euiSelectableListItem__text` element, rather than using the `append` prop. If that doesn't work, you could also try styling a parent container of the badge and icon elements to occupy the full space between the selectable text and the focus badge. Just let me know if you want to chat about it.

Reply:
![Mar-15-2024 10-10-16](https://github.com/elastic/kibana/assets/4283304/fd6b9eff-2197-493f-9b92-fc294f869882)
Fixed it with styles manipulation, we cannot pass node to the text element you mention, just a string.